lib/rules/base.js

```diff
@@ -83,10 +83,10 @@ module.exports = class Base {
     return config;
   }
 
-  getVisitor(env) {
+  async getVisitor(env) {
```

Comment:
Should remove `env` argument completely

```suggestion
  async getVisitor() {
```